### DISCUSSION TITLE
Home Assistant OS 13.2.rc1

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3598] [Announcements]
## Home Assistant Operating System

* Fix guest-agent not starting with HyperV Enlightenments enabled (#3592) @sairon
* Bump Buildroot to 2024.02.6 (#3586) @sairon
* Relocate HAOS Systemd drop-ins to /usr/lib/systemd (#3582) @sairon
* Bump Docker to v27.2.0 (#3576) @sairon
* Bump Buildroot to 2024.02.5 (#3550) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.52 (#3591) @sairon

## Open Virtual Appliance

* Fix guest-agent not starting with HyperV Enlightenments enabled (#3592) @sairon
* Linux: Update kernel to 6.6.52 (#3591) @sairon
* Enable debug logging for GRUB rescue shell options (#3581) @sairon
* Fix missing Intel AX210 firmware files, remove unused firmwares (#3556) @sairon

## Generic x86-64

* Use GRUB2 legacy loader for Macbook Air A1370 (#3593) @sairon
* Linux: Update kernel to 6.6.52 (#3591) @sairon
* Enable debug logging for GRUB rescue shell options (#3581) @sairon
* Use GRUB legacy loader for HP t520 to fix boot with GRUB 2.12 (#3562) @sairon
* Fix missing Intel AX210 firmware files, remove unused firmwares (#3556) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.52 (#3591) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.52 (#3591) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.52 (#3591) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.52 (#3591) @sairon
* Enable debug logging for GRUB rescue shell options (#3581) @sairon
* Fix missing Intel AX210 firmware files, remove unused firmwares (#3556) @sairon

## Build System

* Add test for no failed Systemd units (#3583) @sairon
* Tests: update labgrid to v24.0.1 (#3584) @sairon
* Purge Cloudflare cache for all files after uploading R2 artifacts (#3568) @sairon

## Changes

* Update buildroot branch in .gitmodules to 2024.02.x-haos (#3589) @adeepn

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/13.2.rc1'>Home Assistant OS 13.2.rc1</a>.</em>

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.
